GitUI/UserControls/FileStatusList.cs

```diff
@@ -1431,6 +1535,32 @@ private void FileStatusListView_ContextMenu_Opening(object sender, CancelEventAr
             }
         }
 
+        public void SearchCommit_Click(string text)
+        {
+            if (!SearchEnabledForList)
+            {
+                return;
+            }
+
+            SearchCommitForm search = new()
```

Comment:
```suggestion
            using SearchCommitForm search = new()
```

Reply:
The form lives after this section. Will block multiple opening though